app/scripts/directives/nav.js

```diff
@@ -329,8 +329,23 @@ angular.module('openshiftConsole')
             });
 
```

Comment:
Pulled & ran it to see if I could replicate the bug.  Haven't quite figured out how this change fixes the test, but +1 😄